app/helpers/pages_helper.rb

```diff
@@ -27,22 +27,15 @@ def page_slug(site_label, page_type_url, slug)
     ].join('/').html_safe
   end
 
-  def render_form_for_layout(site, page)
-    layout = page.layout
-    form_url = site_page_path(site, page)
-
-    if lookup_context.template_exists? "pages/_form_for_#{layout.identifier}"
-      render "form_for_#{layout.identifier}", form_url: form_url
-    else
-      render 'form', form_url: form_url
-    end
-  end
-
   def display_additional_button_menu?(page, user)
     return false if user.editor? || page_state_buttons(page).empty?
     page.publishable?
   end
 
+  def display_metadata_form_fields?(page)
+    page.layout.identifier != 'home_page' && page.layout.identifier != 'footer'
```

Comment:
Can you change to an array? We are adding more layouts that have different forms and an array will be ideal. E.g: `!page.layout.identifier.in?(%w(home_page footer))`